src/java/org/apache/cassandra/service/accord/AccordKeyspace.java

```diff
@@ -652,8 +646,9 @@ public static Command loadCommand(AccordCommandStore commandStore, TxnId txnId)
                     return Command.SerializerSupport.committed(attributes, status, executeAt, promised, accepted, waitingOnCommit, waitingOnApply);
                 case PreApplied:
                 case Applied:
+                    return Command.SerializerSupport.executed(attributes, status, executeAt, promised, accepted, waitingOnCommit, waitingOnApply, writes, Result.APPLIED);
```

Comment:
Nobody does anything w/ this right now anyway, but I don't know if we need to distinguish between pre-applied and applied...

Reply:
`org.apache.cassandra.service.accord.AccordObjectSizes#command` needs to be updated right?